bot.py

```diff
@@ -43,6 +44,53 @@ async def on_member_join(member):
     await member.dm_channel.send(f"Hi {member.name}, welcome to 관전남 월드!")
 
 
+# remove default help command to use our help command
+bot.remove_command("help")
```

Comment:
why is this needed? what happens if you don't remove it?

Reply:
if we do not remove the default help command, an error will occur. 
I wil attach the screenshot of this error.
<img width="940" alt="Screen Shot 2021-07-02 at 1 34 56 AM" src="https://user-images.githubusercontent.com/47313362/124225719-ea604900-dad5-11eb-8001-882a32703376.png">
The error said `The command help is already an existing command or alias.`